File: 백준 Study/반복문/num8393.py
# 백준 8393번 n이 주어졌을 때, 1부터 n까지 합을 구하는 프로그램을 작성하시오.
n = int(input())
i=1
sum=0
for i in range (1, n+1) :
    sum+=i
    i+=1

print(sum)    
    

File: 백준 Study/반복문/num8393.java
// import java.util.Scanner;

// public class num8393 {
//     public static void main(String[] args) {
//         Scanner sc = new Scanner(System.in);
//         int num = sc.nextInt();
//         int sum=0;
//         sc.close();
//         for(int i=1; i<=num; i++) {
//             sum+=i;
//         }
//         System.out.print(sum);
//     }
// }
import java.io.BufferedReader;
import java.io.InputStreamReader;
import java.io.IOException;

public class num8393 {
    public static void main(String[] args) throws IOException {
        BufferedReader bf = new BufferedReader(new InputStreamReader(System.in));
        int num = Integer.parseInt(bf.readLine());
        int sum=0;
        for(int i=1; i<=num; i++) {
            sum+=i;
        }
        System.out.print(sum);
    }
}